packages/at_secondary_server/lib/src/verb/handler/stats_verb_handler.dart

```diff
@@ -108,12 +111,21 @@ class StatsVerbHandler extends AbstractVerbHandler {
     return stats;
   }
 
-  Future<void> addStatToResult(id, result) async {
+  Future<void> addStatToResult(
+      id, result, List<String> enrolledNamespaces) async {
     logger.info('addStatToResult for id : $id, regex: $_regex');
     var metric = _getMetrics(id);
     var name = metric.name!.getName();
     dynamic value;
-    if ((id == '3' || id == '15') && _regex != null) {
+    if (id == '3') {
+      if (_regex == null || _regex.isEmpty) {
+        _regex = '.*';
+      }
+      value = await (metric.name as LastCommitIDMetricImpl)
+          .getMetrics(regex: _regex, enrolledNamespaces: enrolledNamespaces);
+    } else if (id == '15' && _regex != null) {
+      // When connection is authenticated via the APKAM, return the highest commit-Id
+      // among the specified namespaces.
```

Comment:
Shouldn't these two comment lines should be in the `id == '3'` block above?

Reply:
Moved the code comments above.